cypress-io/cypress
# chore: show correct cloud project title in notifications

- Closes https://github.com/cypress-io/cypress/issues/27089

### Additional details


Show the Cloud Project name in the notification. Previously it was the local project, based on the directory name, which is incorrect.

We did not have a central place to fetch and collect cloud project metadata, so I added one. I also made a small refactor - we were duplicating `operation` and `operationDoc` in many GraphQL remote requests, so I collapsed them to use a single property, `operationDoc`. `operation` is derived from `operationDoc` via `print` from the `graphql` package.

### Steps to test


1. create any project, ensure directory name is NOT the same as cloud project name
2. trigger a run
3. observe notification has the correct cloud project name, not the directory based name

### How has the user experience changed?


Comply with requirements in the initiative: https://app.clickup.com/18033298/docs/h6amj-43107/h6amj-41927?block=block-7032488f-6f28-4593-a428-dbb17f12c0cd

### PR Tasks
